examples/notebooks/show_remote_cubes.py

```diff
@@ -29,10 +30,16 @@ def show_remote_cubes(bucket, endpoint_url, region_name):
                 ed = pd.to_datetime(str(ds.time.values[-1]))
                 end_date = ed.strftime('%Y-%m-%d')
                 chunksize = ds[var_list[0]].data.chunksize
+                try:
+                    spat_cov = (ds.attrs['geospatial_lon_min'], ds.attrs['geospatial_lat_min'],
+                                ds.attrs['geospatial_lon_max'], ds.attrs['geospatial_lat_max'])
+                except KeyError:
+                    spat_cov = None
                 df = df.append({'cube_name': filename,
                                 'chunks (time, lat, lon)': chunksize,
```

Comment:
See comment above.

Unfortunately, there is currently no way to add metadata to `DataFrame` series, e.g. `df.chunks.attrs['description'] = 'chunk sizes (time, lat, lon)'`